models/_sector/referral/rewards/_schema.yml

```diff
@@ -0,0 +1,74 @@
+version: 2
+
+models:
+
+  - name: referral_rewards
+    meta:
+      blockchain: ethereum, solana, bnb, optimism, arbitrum, polygon
+      sector: nft
+      contributors: soispoke, hildobby, ilemi, 0xRob, cat, umer_h_adil
+    config:
+      tags: ['nft', 'opensea', 'looksrare', 'x2y2', 'magiceden', 'sudoswap', 'foundation', 'element', 'zora', 'ethereum', 'bnb', 'solana', 'events', 'polygon', 'optimism', 'arbitrum', 'rarible', 'aavegotchi', 'oneplanet', 'fractal']
+    description: >
+        NFT mints
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - project
+            - tx_hash
+            - sub_tx_id
+    columns:
+      - name: blockchain
+        description: "Blockchain"
+      - name: project
+        description: "Project"
+        tests:
+          - not_null
```

Comment:
nice, i assume this goes towards your point to run this test and then have it available for the new `sector.info` relationship tests

Reply:
I'm trying to get in the habit of testing `not_null` on all columns that are used in the `unique_key`. (as nulls will produce duplicates in the merge stap).

But yeah this could also link to a `referrals.info` table :+1: